model/src/pyrenew/transformation/builtin.py

```diff
@@ -27,5 +28,5 @@ def ScaledLogitTransform(
         - numpyro.distributions.transforms.SigmoidTransform().inv
     """
     return nt.ComposeTransform(
-        [nt.AffineTransform(0.0, 1.0 / x_max), nt.SigmoidTransform().inv]
+        [nt.SigmoidTransform().inv, nt.AffineTransform(0.0, 1.0 / x_max)]
```

Comment:
I don't think this should be reversed.

Reply:
From DMs: _Had gone in, thinking that perhaps was incorrect but example showed otherwise but didn't change it back before I committed and haven't made an updated commit yet_